src/scss/_themes.scss

```diff
@@ -63,7 +66,7 @@
 				padding: 0 10px;
 			};
 
-			&:nth-child(4) {
+			&:last-child {
```

Comment:
Ah, but I thought that was designed to have rows of `4` always. And if we change it to rows of `3` etc. this component would change?